docs/guide/serialization.rst

```diff
@@ -0,0 +1,310 @@
+Serializing GUFE objects
+========================
+
+When you create objects the follow the GUFE APIs, you will want them to be
+serializable; that is, you'll want to be able to store everything to disk.
+In this section, we'll discuss how GUFE handles serialization, and what a
+developer needs to do to serialize an object.
+
+From a user perspective, this information is mainly relevant for
```

Comment:
There's not a lot in these docs for users (as opposed to developers).  I think users would be better served with an abridged version of these docs somewhere else (openfe docs maybe) else there's a lot of skipping over pieces.  So maybe this little stub of hinting that it might be useful for users to read this can go.

And then `gufe` is the developer package right?  So all of `guide/` is just dev docs, so the content is at home here